cln/cln_client.go

```diff
@@ -121,10 +116,11 @@ func (c *ClnClient) OpenChannel(req *lightning.OpenChannelRequest) (*wire.OutPoi
 		!req.IsPrivate,
 		minConfs,
 		glightning.NewMsat(0),
-		minDepth,
-		glightning.NewSat(0),
```

Comment:
We can't remove the 0 reserve parameter here. If it's not available, we'll need to fork glightning from the elementsproject/glightning version.